cmd/placement/main.go

```diff
@@ -94,15 +97,15 @@ func main() {
 	}
 
 	p := placement.NewPlacementService()
-	go p.Run(*port, certChain)
+	go p.Run(fmt.Sprint(*placementPort), certChain)
```

Comment:
maybe `strconv.Itoa(*placementPort)` is better than `fmt.Sprint` because we do not use any format string.

Reply:
Changed.